### PR TITLE
Get url

### LINKED ISSUE #17: Access the URLs that the user is viewing on the extension
# What is it
See what it takes to access the URLs the user is viewing while the user has the extension active

## Details
Check the [stackoverflow post here](https://stackoverflow.com/questions/1979583/how-can-i-get-the-url-of-the-current-tab-from-a-google-chrome-extension)

## Deliverables
A PR modifying the extension to read the URL of the tab the user is viewing, and for now, simply do an alert or log the URL, we'll have followup issues to use it

### PR DESCRIPTION
**Pull request by MayKikuchi**
We can get the user's url using the content.js script or the background.js script. I've implemented both for now.

To use:
- pull the changes and reload the extension in chrome://extensions.
- there will be two buttons in the extension popup. Click the first button to get the h1 text and the current url (from content.js) or click the second button to get the url (from background.js). They should be displayed as lines of text in the popup.

Also, we can restrict our extension to only do things when on certain sites by checking the url with regex (there is an example in the background.js file) or we can restrict where the content.js file should work in the manifest.json using the "matches" key under "content_scripts". 

closes #17

### CODE REVIEW

**Comment by MayKikuchi** [comment]
whoops found a bug in the background script :P

**Comment by MayKikuchi** [comment]
^fixed a bug and re-opened (idk why the timeline has the closing after the reopening :P)

File: frontend/background.js [hunk 1]
```diff
@@ -1 +1,44 @@
 console.log("background.js is executed when extension is installed or refreshed");
+
+const listOfInjectableURLRegex = [ // change these to the list of allowed news sources
```
Comment (tarikeshaq) [inline frontend/background.js:3]: Thank you for this!!! 🚀

**Review by tarikeshaq — APPROVED** [review]
This looks great! Lots of good work @MayKikuchi! I'm very happy with this! Looks like we might start styling stuff next week 👀

**Pull request merged**